.jane-openapi.php

```diff
@@ -13,4 +13,7 @@
     'reference' => true,
     'strict' => false,
     'client' => 'psr18',
+    'clean-generated' => true,
```

Comment:
default value is `true`, this option is not usefull here

Reply:
I prefer explicitness here :wink: